tests/tests/test_timetable.py

```diff
@@ -247,3 +247,42 @@ def _get_train_schedule_simulation_response(
     sim_response.raise_for_status()
     content = sim_response.json()
     return content
+
+
+def create_and_delete_paced_train(timetable_id: int):
+
+    payload = [
+        {
+            "comfort": "STANDARD",
+            "constraint_distribution": "STANDARD",
+            "initial_speed": 0,
+            "labels": [],
+            "options": {"use_electrical_profiles": False},
+            "path": [
+                {"id": "start", "track": "TC0", "offset": 185000},
+                {"id": "stop", "track": "TC0", "offset": 685000},
+                {"id": "end", "track": "TD0", "offset": 24820000},
+            ],
+            "power_restrictions": [],
+            "rolling_stock_name": "fast_rolling_stock",
+            "schedule": [
+                {
+                    "at": "start",
+                },
+                {"at": "stop"},
+                {
+                    "at": "end",
+                },
+            ],
+            "speed_limit_tag": "MA100",
+            "start_time": "2024-05-22T08:00:00.000Z",
+            "train_name": "with_stop",
+            "duration": "PT2H",
+            "step": "PT15M",
+        }
+    ]
+    pt_response = requests.post(f"{EDITOAST_URL}/timetable/{timetable_id}/paced_train", json=payload)
+    pt_response.raise_for_status()
+    paced_train_id = pt_response.json()[0]["id"]
+    r = requests.delete(f"{EDITOAST_URL}paced_train/", json={"ids": [paced_train_id]})
+    r.raise_for_status()
```

Comment:
If a `GET /paced_train/<id>` endpoint were to exist, you could actually verify that the deletion worked.

Reply:
You are write, we will write the integration tests in another PR when all the endpoints will be written to test all cases. Wdyt?